pkg/operator/ceph/config/network.go

```diff
@@ -50,10 +50,8 @@ func generateNetworkSettings(clusterdContext *clusterd.Context, namespace string
 	cephNetworks := []Option{}
 
 	for _, selectorKey := range NetworkSelectors {
-		// This means only "public" was specified and thus we use the same subnet for cluster too
+		// When only one network selector is specified
```

Comment:
I still don't really understand this comment. I think it's safe to remove, tbh. All this code does is skip processing selectors if they aren't specified.

Reply:
Changing the comment